delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/AclsRoutesSpec.scala

```diff
@@ -67,9 +70,12 @@ class AclsRoutesSpec
   val myOrgMyProj2  = Project(Label.unsafe("myorg"), Label.unsafe("myproj2"))
   val myOrg2MyProj2 = Project(Label.unsafe("myorg2"), Label.unsafe("myproj2"))
 
-  private val acls = AclsDummy(
-    PermissionsDummy(Set(aclsPermissions.read, aclsPermissions.write, managePermission, events.read))
-  ).accepted
+  private val acls =
+    (for {
+      perms  <- PermissionsDummy(Set(aclsPermissions.read, aclsPermissions.write, managePermission, events.read))
```

Comment:
We could use AclSetup for this part too no ?
There could be another method there to init all this